angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts

```diff
@@ -0,0 +1,1974 @@
+  `,
+});
+
+export const mls = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="mls"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+mls.story = {
+  name: 'MLS',
+};
+
+export const mobile = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="mobile"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const mortgage = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="mortgage"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const multiFamily = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="multi-family"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+multiFamily.story = {
+  name: 'Multi-Family',
+};
+
+export const multiplePages = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="multiple-pages"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const oneTimeUpdate = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="one-time-update"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const organize = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="organize"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const organizeFilled = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="organize-filled"
+      additionalClasses="sprk-c-Icon--xl sprk-c-Icon--filled"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const organizeFilledSmall = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="organize-filled-small"
+      additionalClasses="sprk-c-Icon--filled"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const ourFamily = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="our-family"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const parking = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="parking"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const payment = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="payment"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const paymentCenter = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="payment-center"
+      additionalClasses="sprk-c-Icon--xl"
+    >
+    </sprk-icon>
+  `,
+});
+
+export const percent = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-icon
+      iconType="percent"
+      additionalClasses="sprk-c-Icon--xl sprk-c-Icon--filled"
```

Comment:
in our docs we have this as `<sprk-icon iconType="percent" additionalClasses="sprk-c-Icon--l"></sprk-icon>
` without the filled class

Reply:
Same as the dollar sign.